BlockchainSdk/Common/Interfaces/TransactionValidator/FeeResourceRestrictable.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  FeeResourceRestrictable.swift
+//  BlockchainSdk
+//
+//  Created by Aleksei Muraveinik on 06.06.24.
+//  Copyright © 2024 Tangem AG. All rights reserved.
+//
+
+import Foundation
+
+protocol FeeResourceRestrictable {
+    func validateFeeResource(amount: Amount, fee: Amount) async throws
```

Comment:
`async` не нужен здесь 
 ```suggestion
    func validateFeeResource(amount: Amount, fee: Amount) throws
```

Reply:
https://github.com/tangem/blockchain-sdk-swift/pull/703#discussion_r1639754878

Я в замешательстве. Как в итоге?

Reply:
А, это другое

Reply:
`async` нужен только для `async` кода, запросы, делеи ))
а тут же все синхронно выполняется